sdk/keyvault/azure.keyvault.keys.txt

```diff
@@ -0,0 +1,180 @@
+**** Package/Module azure.keyvault.keys****
+class KeyClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_key(self, name: str, **kwargs) -> bytes: ...
+	def create_ec_key(self, name: str, hsm: bool, curve: curve=None, key_operations: key_operations=None, enabled: bool=None, expires: datetime.datetime=None, not_before: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
```

Comment:
Copy/pasting my personal debate on "enabled" from the secret PR:
I guess "enabled: bool=None" is about not populating the model to send on the wire, but would be clearer IMHO to have "True/False" for real. Unless customer can change the default on the server side, then I can understand it's not possible to take side on the client side. But if server is always "False" by default, I don't see how Azure can possibly change one day, I would suggest to make it clear here.
At worst, update the doc to say "Azure default if nothing is set is False" or whatever the actual default value.

Reply:
From a client library version/breaking change perspective this is probably worth discussing in the broader review. There is a tension between allowing the service to provide the default value and keeping compatibility in the client library. The service behavior *can* change between API versions.

Reply:
We can address this in future releases if the default on the service changes. At that time we can decide if adding a client default would be warranted.